srtcore/packetfilter.cpp

```diff
@@ -168,7 +168,7 @@ bool PacketFilter::packControlPacket(int32_t seq, int kflg, CPacket& w_packet)
     // - Crypto
     // - Message Number
     // will be set to 0/false
-    w_packet.m_iMsgNo = MSGNO_PACKET_BOUNDARY::wrap(PB_SOLO);
+    w_packet.m_iMsgNo = SRT_MSGNO_CONTROL | MSGNO_PACKET_BOUNDARY::wrap(PB_SOLO);
```

Comment:
What is the purpose of `SRT_MSGNO_CONTRO = 0` here?

Reply:
This is the actual message number that is set here. Before this number was simply set "implicitly" because the value assigned contained only set bits in the range for `MSGNO_PACKET_BOUNDARY` and all others - including those represending the msgno - were clear. This constants declares explicitly that we do set the message number, and this message number is equal to `SRT_MSGNO_CONTROL`.